This patch is part of a series of patches for festival for
Debian GNU/Linux, to comply with Filesystem Hierarchy Standard

Index: b/lib/soleml-mode.scm
===================================================================
--- a/lib/soleml-mode.scm
+++ b/lib/soleml-mode.scm
@@ -330,7 +330,7 @@
        (list 'exit_func soleml_exit_func)
        '(analysis_type xxml)
        (list 'filter 
-	     (format nil "%s -D %s " sgml_parse_progname libdir))))
+	     (format nil "%s -D %s " sgml_parse_progname datadir))))
     tts_text_modes))
 
 (provide 'soleml-mode)
